Mathlib/SetTheory/Cardinal/Cofinality.lean

```diff
@@ -1199,22 +1199,14 @@ namespace Ordinal
 open Cardinal
 open scoped Ordinal
 
--- TODO: generalize universes
+-- TODO: generalize universes, and use ω₁.
```

Comment:
Why can't you do that right now?

Reply:
I need `ω₁.cof = ℵ₁` for this proof, which requires `ord (ℵ_ o) = ω_ o`, which is nontrivial to prove currently but becomes almost trivial after #17674.